collections/dominode-resources/processing/legacy.py

```diff
@@ -0,0 +1,66 @@
+import typing
+
+from qgis.core import (
+    QgsProcessingAlgorithm,
+    QgsProcessingParameterExpression,
+    QgsProcessingOutputString,
+)
+from qgis.PyQt.QtCore import QCoreApplication
+
+
+class Legacy(QgsProcessingAlgorithm):
+
+    INPUT = 'INPUT'
+    OUTPUT = 'OUTPUT'
+
+    def tr(self, string):
+        return QCoreApplication.translate('Processing', string)
+
+    def createInstance(self):
+        return Legacy()
+
+    def name(self):
+        """
+        Returns the unique algorithm name.
+        """
+        return 'legacy'
+
+    def displayName(self):
+        """
+        Returns the translated algorithm name.
+        """
+        return self.tr('Legacy')
```

Comment:
Please be a bit more verbose on these names to specify clearly what this is (similar to the request for changing the class name above)